src/runtime/utils.lambda.ts

```diff
@@ -1,3 +1,4 @@
+import { isUtf8, Buffer } from "node:buffer";
```

Comment:
`isUTF8` is quite recent. We might need to conditionally use it  https://nodejs.org/api/buffer.html#bufferisutf8input

Reply:
The current issue is that the JSON is a buffer as well, so without the utf8 check the JSON response is base64 encoded.

So we need something that works prior to node 18. 

I've implemented a hacky (?) fallback solution but would it make sense to add a polyfill to unenv for it using https://github.com/hcodes/isutf8/blob/master/src/index.ts make sense?

Reply:
Or do we just use the package https://www.npmjs.com/package/isbinaryfile

Reply:
outgoing body wouldn't be JSON. h3 already handles that. I think we can remove the checks altogether. (body is either string or buffer at that stage) Can you please double check?

Reply:
Without the checks, 4 of the tests fail with this type of response

```
 FAIL  test/presets/aws-lambda.test.ts > nitro:preset:aws-lambda > handles errors
AssertionError: expected 'SGV5IEFQSQ==' to contain 'Hey API'
 ❯ test/tests.ts:249:29
    247|     expect(status).toBe(503);
    248|     const { data: heyData } = await callHandler({ url: "/api/hey" });
    249|     expect(heyData).to.have.string("Hey API");
       |                             ^
    250|   });
```

When inspecting the output using the following code

```ts
if (Buffer.isBuffer(body)) {
    console.log(typeof body, body.toString())
 // ...
```

The following is shown

```
object {"message":"Hello API"}
object Hey API
object Works!
```

Not sure if this is a testing artifact? As I would have thought JSON would be returned as a string as well.

Reply:
Yeat it seems localCall already wraps to Buffer. Thinking to somehow improve that area to reduce workloads here.

Reply:
(still working on it between meetings) Exciting finding after digging some holes! The two `/api/hey` and `/api/hello` fail because, during tests, we prerender those two, and then when trying to fetch, serve static middleware handles the response and sends the response as a buffer instead!) i am thinking of some root cause fixes to follow up.

Reply:
I think safest would be to check about known text types for Buffer edge case. Later we might also improve serve static to avoid buffer conversion for text types.